### PR TITLE
(#3386) Update Error Pages

### LINKED ISSUE #3386: Story: Update Error Pages for NCIDS Layout
## As a user, I would like to view Error Page content using the NCIDS, so that I have a better navigation experience.

This ticket is to update the error page template which uses a full-page layout without left navigation.

As part of this ticket, the Inner Pages background color should be updated to be #FFFFFF

> **ESTIMATE** 3

### Acceptance Criteria

```gherkin
Scenario: User goes to a non-existent page and the error page displays
  When a user navigates to www.cancer.gov/foo
  Then the error page displays
    And the page displays the updated NCIDS Header
    And the page displays the updated NCIDS Footer
    And the page does not display the NCIDS Side Nav
```

### Resources:
* Feature: JIRA Ticket #
* Mockups: *Here goes a URL to or the name of the mockup(s) in inVision*
* Web Page: *Here goes a URL to an existing web page*

### Notes

*Some complementary notes if necessary:*

* > Here goes a quote from an email
* Here goes whatever useful information can exist…

---

### Prerequisites
- *required dependency for identifying the problem, scope and then solution for this issue*
- [ ] Short Spike Description - Issue #9999

### Tasks
- [ ] Short Task Description - Issue #9999
- [ ] Short Task Description - Issue #9999
  - https://www.cancer.gov/foo
  - https://www.cancer.gov/espanol/foo
  - https://dceg.cancer.gov/foo

### Technical Notes
- More notes, or technical nuances/direction

### PR DESCRIPTION
**Pull request by dev-rana-publicis**
ODE: http://ncigovcdode501.prod.acquia-sites.com/
Closes #3386 

Check Error Pages at: 
https://ncigovcdode501.prod.acquia-sites.com/ErrorPages/404/
https://ncigovcdode501.prod.acquia-sites.com/espanol/ErrorPages/404/

### CODE REVIEW

**Review by karlikpj — APPROVED** [review]

**Review by olitharp-nci — APPROVED** [review]

**Comment by bennettcc** [comment]
Per slack conversion on 9/9, this has passed product/UX review.

Awaiting final tech review approval from @bryanpizzillo @arcepaul before it can be merged

**Review by arcepaul — APPROVED** [review]

File: docroot/profiles/custom/cgov_site/themes/custom/cgov/src/styles/_errorPage.scss
Comment (bryanpizzillo) [inline docroot/profiles/custom/cgov_site/themes/custom/cgov/src/styles/_errorPage.scss:4]: Hrmm. Can we do this not in this file. Changing this file will change DCEG Connect. If it is simple enough you can move it to ncids_trans/front-end/legacy/styles (just follow the convention). Then import it from ncids_global. OR, we could make a new entry point for it, and move this into legacy.

**Review by bryanpizzillo — CHANGES_REQUESTED** [review]

**Review by bryanpizzillo — APPROVED** [review]

**Pull request merged**
